src/txn/txn.c

```diff
@@ -964,32 +953,30 @@ __txn_fixup_prepared_update(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor, WT_U
      */
     F_SET(txn, WT_TXN_PREPARE_IGNORE_API_CHECK);
 
+    /*
+     * Set the stop time point to be the committing transaction's time point and copy the start time
+     * point from the current history store update.
+     */
     tw.stop_ts = txn->commit_timestamp;
     tw.durable_stop_ts = txn->durable_timestamp;
     tw.stop_txn = txn->id;
-    WT_TIME_WINDOW_SET_START(&tw, fix_upd);
+    tw.start_ts = hs_tw->start_ts;
```

Comment:
It feels like maybe we need to expand the set of time window initializers? Ideally there would be a single call with the start coming from hs_tw and stop coming from txn.
I don't think it needs to be a macro - could be an inline function.

Reply:
Added a macro to do that.